File: NERP.NetworkEntities/Models/SpliceClosure.cs
using NERP.Common.Models;

namespace NERP.NetworkEntities.Models
{
    public class SpliceClosure : BaseNetworkEntity
    {
        public Point Geometry { get; set; }
        public int? NoOfPorts { get; set; }
        public Installation Installation { get; set; }
        public Owner Owner { get; set; }
    }
}

File: NERP.Admin.Web1/ClientApp/src/app/entities/models/spliceClosure.ts
import { Point } from '../../common/geo/geometry';
import { BaseNetworkEntity } from './baseNetworkEntity';
import { Installation } from './installation';
import { Owner } from './owner';

export class SpliceClosure extends BaseNetworkEntity {
    noOfPorts?: number;
    geometry: Point;
    installation?: Installation = new Installation();
    owner?: Owner = new Owner();
}
